src/main/java/appeng/crafting/execution/ElapsedTimeTracker.java

```diff
@@ -77,10 +81,10 @@ public long getElapsedTime() {
     }
 
     public long getRemainingItemCount() {
-        return this.remainingItemCount;
+        return Math.round(((float) this.remainingItemCount) / AEKEY_SCALE_FACTOR);
     }
 
     public long getStartItemCount() {
-        return this.startItemCount;
+        return Math.round(((float) this.startItemCount) / AEKEY_SCALE_FACTOR);
```

Comment:
Why not just use integer division here?

Reply:
> Why not just use integer division here?

Integer Division equals Math.floor, making the 99/10 = 9 instead of 10 in this case.
I want to round to make it more precise in displaying, for showing progress, the trade-off is performance.

Reply:
> If you want to simulate math.round() with integer division you can add half of the AE_SCALE_FACTOR before dividing, right?

You are right. I have updated.